lib/cli/scripts/resources.ts

```diff
@@ -19,196 +19,192 @@
-            poolIntermediateMessageCatch: 'pool-interm-message-catch',
-            poolInterruptingBoundarySubprocessThrow: 'pool-int-bound-subpr-throw',
-            poolInterruptingBoundarySubprocessCatch: 'pool-int-bound-subpr-catch',
-            poolInterruptingBoundaryThrow: 'pool-int-boundary-throw',
-            poolInterruptingBoundaryCatch: 'pool-int-boundary-catch',
-            outputVariablesMapping: 'output-variables-mapping'
-        },
-        forms: {
-            tabVisibilityFields: {
-                name: 'tabvisibilitywithfields'
-            },
-            tabVisibilityVars: {
-                name: 'tabvisibilitywithvars'
-            },
-            usertaskform: {
-                name: 'usertaskform'
-            },
-            dropdownform: {
-                name: 'dropdownform'
-            },
-            formVisibility: {
-                name: 'form-visibility'
-            },
-            multilingualform: {
-                name: 'multilingualform'
-            },
-            inputform: {
-                name: 'inputform'
-            },
-            outputform: {
-                name: 'outputform'
-            },
-            exclusiveconditionform: {
-                name: 'exclusive-condition-form'
-            },
-            uploadlocalfileform: {
-                name: 'upload-localfile-form'
-            },
-            booleanvisibility: {
-                name: 'booleanvisibility'
-            },
-            requirednumbervisibility: {
-                name: 'requirednumbervisibility'
-            },
-            mealform: {
-                name: 'mealform'
-            },
-            resultcollectionform: {
-                name: 'resultcollectionform'
-            },
-            uploadSingleMultiple: {
-                name: 'upload-single-multiple',
-                widgets: {
-                    contentMultipleAttachFileId: 'UploadMultipleFileFromContentId'
-                }
-            },
-            formWithJsonWidget: {
-                name: 'form-with-json-widget'
-            },
-            formWithAllWidgets: {
-                name: 'form-with-all-widgets'
-            },
-            poolForm: {
-                name: 'pool-usertaskform'
-            },
-            formWithSingleInput: {
-                name: 'form-with-single-input'
-            }
-        },
-        tasks: {
-            processstring: 'inputtask',
-            uploadSingleMultipleFiles: 'UploadSingleMultipleFiles'
-        },
-        security: [
-            { role: 'APPLICATION_MANAGER', groups: [], users: ['manageruser'] },
-            { role: 'ACTIVITI_ADMIN', groups: [], users: ['superadminuser', 'processadminuser'] },
-            { role: 'ACTIVITI_USER', groups: ['hr', 'sales', 'testgroup'], users: ['hruser'] }
-        ],
-        infrastructure: { connectors: { restconnector: {} }, bridges: {} },
-        enableLocalDevelopment: true
-    },
+    // SUB_PROCESS_APP: {
```

Comment:
I believe either we leave this code or remove it completely.
If you are still testing, then mark the PR as draft, please